### DISCUSSION TITLE
Shell completions

### DISCUSSION THREAD

**Discussion by medihack** [pawamoy/duty#25] [Q&A]
While looking around for a pyinvoke alternative, I stumbled upon duty and it seems like a perfect fit for me. One thing I missed from looking at the documentation is shell completions ([something like in pyinvoke](https://docs.pyinvoke.org/en/stable/invoke.html#shell-tab-completion)). Do you plan to add this?

**Comment by pawamoy** [comment]
Sure! Lets open a feature request.
